src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/Regions/Queries/GetRegions/GetRegionsQueryResult.cs

```diff
@@ -0,0 +1,14 @@
+﻿using SFA.DAS.ApprenticeAan.Application.Entities;
+
+namespace SFA.DAS.ApprenticeAan.Application.Regions.Queries.GetRegions
+{
+    public class GetRegionsQueryResult
```

Comment:
We dont have to have Result object if it does not make sense, like in this case the Response can easily be List<Region>

Reply:
No, I thought that. But we might as well keep it mirrored to AanHub.Api if it's not too much overhead?

Reply:
I am not too fussed, but I would rather avoid any boiler plate code.